vhdbuilder/packer/components.json

```diff
@@ -83,13 +83,19 @@
       "amd64OnlyVersions": [],
       "multiArchVersions": [
         "v1.4.22",
-        "v1.4.27"
+        "v1.4.27",
+        "v1.4.28",
+        "v1.4.29"
       ]
     },
     {
       "downloadURL": "mcr.microsoft.com/containernetworking/azure-npm:*",
       "amd64OnlyVersions": [],
-      "multiArchVersions": []
+      "multiArchVersions": [
+        "v1.4.22",
+        "v1.4.27",
+        "v1.4.29"
```

Comment:
let's only have 1.4.29 in multiArch, since windows will only be using 29+

Reply:
also, might as well not introduce the other two images. 1.4.29 will be the only one in prod soon

Reply:
updated